juniper_official/oam/ethernet-oam-lfm.yml

```diff
@@ -0,0 +1,10 @@
+---
+EvpnLFMTable:
+    rpc: get-lfmd-information
```

Comment:
How big the response of this command be?
If it will have scale problem, Please set XSLT to filter the response at source. Please talk to Vikram about XSLT example

Reply:
I have to analyse some of the use cases and check response.  Meanwhile approve the MR and i will open task for XSLT

Reply:
Please raise EPIC tickets so that this can be tracked and ping the tickets to me.